Sofa/Component/Constraint/Projective/src/sofa/component/constraint/projective/FixedRotationProjectiveConstraint.h

```diff
@@ -65,9 +65,18 @@ class FixedRotationProjectiveConstraint : public core::behavior::ProjectiveConst
 
 
 protected :
-    Data< bool > FixedXRotation; ///< Prevent Rotation around X axis
-    Data< bool > FixedYRotation; ///< Prevent Rotation around Y axis
-    Data< bool > FixedZRotation; ///< Prevent Rotation around Z axis
+    SOFA_ATTRIBUTE_DEPRECATED__RENAME_DATA()
+    Data< bool > FixedXRotation;
+
+    SOFA_ATTRIBUTE_DEPRECATED__RENAME_DATA()
+    Data< bool > FixedYRotation;
+
+    SOFA_ATTRIBUTE_DEPRECATED__RENAME_DATA()
+    Data< bool > FixedZRotation;
+
+    Data< bool > d_FixedXRotation; ///< Prevent Rotation around X axis
+    Data< bool > d_FixedYRotation; ///< Prevent Rotation around Y axis
+    Data< bool > d_FixedZRotation; ///< Prevent Rotation around Z axis
```

Comment:
```suggestion
    Data< bool > d_fixedXRotation; ///< Prevent Rotation around X axis
    Data< bool > d_fixedYRotation; ///< Prevent Rotation around Y axis
    Data< bool > d_fixedZRotation; ///< Prevent Rotation around Z axis
```
The guidelines say that it should be lowerCamelCase ([N4])